### ISSUE TITLE
Autocompletion for modules from PYTHONPATH Manager

### ISSUE THREAD

**Issue by xxblx** [spyder-ide/spyder#3492]
## Description

Hi, 

Autocompletion doesn't work in editor widget for modules from additional paths from PYTHONPATH Manager. It's very sad if need to use virtual environment and impossible to install modules globally in sys.path. 
## How to reproduce
- Create new project with Spyder in /Users/%username%/project called just "project"
- Create new virtualenv in project directory and install some package (I installed vk, but for issue reproducing allowed to install any other packages that doen't installed globally in python's sys.path) 

```
cd /Users/%username%/project 
pyvenv venv
venv/bin/pip install vk
```
- Add /Users/%username%/project/venv/lib/python3.5/site-packages to PYTHONPATH Manager and re-open Spyder. 
- Autocompletion for modules (in this example - vk) which exists only in venv will works in ipython console widget, but don't works in Editor. 

Screenshots of ipython console widget:
http://itmages.ru/image/view/4966936/db0a13ca

http://itmages.ru/image/view/4966935/4c206664
- If I install modules globally 

```
pip3 install vk
```

Autocompletion for vk module will works fine in Editor too. 

Screenshot of editor: http://itmages.ru/image/view/4966941/a35f6571 
## Versions and main components
- spyder 3.0.0
- python 3.5.2 on macOS, python 3.5.1 on linux
- macOS Sierra 10.12 / Fedora 24 x86_64
- PyQt5
- Qt 5.7.0
## Dependencies

```
jedi >=0.8.1     :  0.9.0 (OK)
matplotlib >=1.0 :  1.5.3 (OK)
nbconvert >=4.0  :  4.2.0 (OK)
numpy >=1.7      :  1.11.1 (OK)
pandas >=0.13.1  :  0.18.1 (OK)
pep8 >=0.6       :  1.7.0 (OK)
psutil >=0.3     :  4.3.1 (OK)
pyflakes >=0.6.0 :  1.3.0 (OK)
pygments >=2.0   :  2.1.3 (OK)
pylint >=0.25    :  1.6.4 (OK)
qtconsole >=4.2.0:  4.2.1 (OK)
rope >=0.9.4     :  0.9.4-1 (OK)
sphinx >=0.6.6   :  1.5a1 (OK)
sympy >=0.7.3    :  1.0 (OK)
```

**Comment by ccordoba12** [comment]
We're going to allow users to associate a virtual or conda environment with a given project. I think when we do that, this problem will be solved.

**Comment by xxblx** [comment]
Sounds good. Thanks!

**Comment by ccordoba12** [comment]
This was fixed in Spyder **3.1.4**.
